File: Archive/MyLocations/MyLocations/MyTabBarController.m
//
//  MyTabBarController.m
//  MyLocations
//
//  Created by Hector Enrique Gomez Morales on 7/3/14.
//  Copyright (c) 2014 Hector Enrique Gomez Morales. All rights reserved.
//

#import "MyTabBarController.h"

@interface MyTabBarController ()

@end

@implementation MyTabBarController

- (UIStatusBarStyle)preferredStatusBarStyle
{
    return UIStatusBarStyleLightContent;
}

- (UIViewController *)childViewControllerForStatusBarStyle
{
    return nil;
}

@end

File: MyLocations/MyLocations/MyTabBarController.swift
//
//  MyTabBarController.swift
//  MyLocations
//
//  Created by Hector Enrique Gomez Morales on 2/11/15.
//  Copyright (c) 2015 Hector Enrique Gomez Morales. All rights reserved.
//

import UIKit

class MyTabBarController: UITabBarController {
    override func preferredStatusBarStyle() -> UIStatusBarStyle {
        return .LightContent
    }
    
    override func childViewControllerForStatusBarStyle() -> UIViewController? {
        return nil
    }
}